cli/package.json

```diff
@@ -45,6 +45,7 @@
     "@types/node": "*",
     "prettier": "2.5.0",
     "ts-node": "10.4.0",
+    "type-fest": "2.8.0",
```

Comment:
f all were using this for is the PackageJson type, we could just [copy it over](https://github.com/sindresorhus/type-fest/blob/main/source/package-json.d.ts) directly instead of adding another dep.

Reply:
We use this package in many of our other packages.